### PR TITLE
gyroflow: Add version 1.0.1

### LINKED ISSUE #7913: [Request] Add gyroflow
### Information

**Name:** Gyroflow

**Description:** Software for video stabilization.

**Homepage:** https://gyroflow.xyz/, https://github.com/gyroflow/gyroflow

**Download link(s):** [Releases page](https://github.com/gyroflow/gyroflow/releases), [Windows 64-bit release](https://github.com/gyroflow/gyroflow/releases/download/v1.0.0-rc1/Gyroflow-windows64.zip)

**Some indication of popularity/repute:** 273 stars on aforementioned repository as of writing, plus positive reception in [Reddit](https://www.reddit.com/r/rust/comments/skbvqx/announcing_gyroflow_an_advanced_video/). Rewritten in Rust from the original Python code, see legacy repository [here](https://github.com/ElvinC/gyroflow).

### PR DESCRIPTION
**Pull request by KnotUntied**
Closes #7913 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
This is a release candidate? Note that Main and Extras buckets only have stable releases. Release candidates go in Versions bucket.

**Comment by KnotUntied** [comment]
My mistake. I guess this can wait until Gyroflow gets a stable release.

**Comment by rashil2000** [comment]
You can put it in the official _Versions_ bucket in the meanwhile.

File: bucket/gyroflow.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "description": "Gyro-assisted video stabilization software.",
+    "homepage": "https://gyroflow.xyz/",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/gyroflow/gyroflow/releases/download/v1.0.1/Gyroflow-windows64.zip",
+            "hash": "ec35451b552e9665624aa46e5b85a21f283ed533efc335e9ab8b7476adb22888"
+        }
+    },
+    "bin": "Gyroflow.exe",
```
Comment (rashil2000) [inline bucket/gyroflow.json:12]: CLI args?

**Review by rashil2000 — APPROVED** [review]

File: bucket/gyroflow.json (hunk 1 above)
Comment (issaclin32) [reply bucket/gyroflow.json:12]: doesn't look like so.
https://github.com/ElvinC/gyroflow/blob/master/gyroflow.py

**Pull request merged**